dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/HTTPConduit.java

```diff
@@ -1506,7 +1506,7 @@ protected boolean processRetransmit() throws IOException {
             case HttpURLConnection.HTTP_MOVED_TEMP:
             case HttpURLConnection.HTTP_SEE_OTHER:
             case 307:
-            case 308:
+//            case 308: Liberty change: line is removed
```

Comment:
I think we should undo all of these changes. HTTPConduit is vastly different now then it was in 2.6.2, and I don't think any of what was done originally for jaxws-2.2 sis still applicable.